silk/views/filterable_requests_view.py

```diff
@@ -0,0 +1,102 @@
+try:
+    # Django>=1.8
```

Comment:
As of fa1f5425dfd811d200f20856ba195fb8062bc491, django-silk does not support django<1.11

Reply:
Ah good, improves this code!